recipes/urlobject/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "URLObject" %}
+{%set version = "2.4.2" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "dd0f08a81d89485b1e587fd60aec4b8482859788e5ace67dd1b0d4a001515c57" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - urlobject
+
+about:
+  home: https://github.com/zacharyvoase/urlobject
+  license: PUBLIC-DOMAIN
+  license_family: PUBLIC-DOMAIN
```

Comment:
Feel free to just drop `license_family`. Seems to cause more issues trying to fix this than just leaving it out.